Dockerfile

```diff
@@ -38,6 +38,14 @@ COPY --chown=mendix:root $BUILD_PATH /opt/mendix/build
 # Add the buildpack modules
 ENV PYTHONPATH "/opt/mendix/buildpack/lib/"
 
+# Work around for Git issue DES-1880
+# 1. Setting PATH variable
+# 2. Creating a binary named 'git'
+# 3. Provding exec permission to 'git' binary
+ENV PATH="/opt/mendix/buildpack/bin:${PATH}"
```

Comment:
Can you add `git` script as a file to `scripts/`, copy/chown it together with `scripts/compilation` and add it to `$PATH` in `scripts/compilation` instead?

Something like:
```
# Copy python scripts which execute the buildpack (exporting the VCAP variables)
COPY --chown=mendix:root scripts/compilation scripts/git /opt/mendix/buildpack
```

```
RUN mkdir -p /tmp/buildcache &&\
    chmod +rx /opt/mendix/buildpack/compilation /opt/mendix/buildpack/git &&\
...
```

and in add it to `$PATH` in `compilation`:

```
    os.environ['VCAP_SERVICES'] = vcap_services_str
    os.environ["PATH"] += os.pathsep + os.pathsep.join('/opt/mendix/buildpack')
```
(https://stackoverflow.com/questions/1681208/python-platform-independent-way-to-modify-path-environment-variable)

This way, no additional layers will be added to the Dockerfile.

Reply:
Have made changes according to above comments.
Kindly review.